### ISSUE TITLE
qase-mocha-reporter and qase-webdriverio-reporter

### ISSUE THREAD

**Issue by jamesmortensen** [qase-tms/qase-javascript#271]
Are there any plans to implement the following reporters:

- qase-mocha-reporter
- qase-webdriverio-reporter ( or wdio-qase-reporter)

Thanks,
James

**Comment by 1ivliev** [comment]
@jamesmortensen Hi. You can place a feature request on our roadmap site:
https://roadmap.qase.io

**Comment by yarinsa** [comment]
> Are there any plans to implement the following reporters:
> 
> 
> 
> - qase-mocha-reporter
> 
> - qase-webdriverio-reporter ( or wdio-qase-reporter)
> 
> 
> 
> Thanks,
> 
> James

Did you ever find solution or wrote custom reporter?

**Comment by gibiw** [comment]
We have released a beta version of the reporter for [Mocha](https://github.com/qase-tms/qase-javascript/tree/main/qase-mocha). We will be glad to receive any feedback

**Comment by gibiw** [comment]
We have released a beta version of the reporter for [WDIO](https://github.com/qase-tms/qase-javascript/tree/main/qase-wdio#readme)

**Comment by YonKu0** [comment]
> We have released a beta version of the reporter for [WDIO](https://github.com/qase-tms/qase-javascript/tree/main/qase-wdio#readme)

Hi Dmitry,

Firstly, great job on the new feature! I've tested it and integrated it into our mobile automation project. While most things are working as expected, I encountered a critical issue during the test run.

Failed tests are not being sent to the test run on the Qase platform. I'd be happy to provide any additional details to help investigate and resolve this issue. If there's a bug, we might be able to address it quickly.

Looking forward to your thoughts!
